site/docs/4.1/components/carousel.md

```diff
@@ -12,6 +12,8 @@ The carousel is a slideshow for cycling through a series of content, built with
 
 In browsers where the [Page Visibility API](https://www.w3.org/TR/page-visibility/) is supported, the carousel will avoid sliding when the webpage is not visible to the user (such as when the browser tab is inactive, the browser window is minimized, etc.).
 
+The carousel can handle swipe events (left and right) only if you include [HammerJS]({{ site.cdn.hammer }} before Bootstrap.
```

Comment:
perhaps reword this a tad:

"The carousel supports swipe gestures (left and right) using <a href="https://hammerjs.github.io/">HammerJS</a>. For this to function correctly you need to include [HammerJS]...."

Reply:
I changed that, thanks Patrick !